routers/home.go

```diff
@@ -155,9 +136,7 @@ func ExploreRepos(ctx *context.Context) {
 	ctx.Data["PageIsExploreRepositories"] = true
 
 	RenderRepoSearch(ctx, &RepoSearchOptions{
-		Ranger:   models.GetRecentUpdatedRepositories,
 		PageSize: setting.UI.ExplorePagingNum,
-		Searcher: ctx.User,
```

Comment:
Why is this line removed?

Reply:
Not needed anymore. But now I see I have to make some changes as `models.GetRecentUpdatedRepositories()` need only `Searcher` and not `OwnerID`.

Reply:
@ethantkoenig I looked at it again and I don't need to change anything. Reason is to unify behaviour when you search with/without keyword. I'll post images.